Berksfile

```diff
@@ -0,0 +1,18 @@
+source 'https://supermarket.chef.io'
+
+cookbook 'aliases',
+         git: 'git@github.com:osuosl-cookbooks/aliases'
```

Comment:
This should fit within 80char. No need to add it down with an intent. The only one that is too long I think is resource_from_hash. If you look at the osl-haproxy cookbook you'll see that format works with rubocop. Mind updating it to be more like that so we're standardized?